bin/runtime/src/chain_extension/baby-liminal-extension/lib.rs

```diff
@@ -74,27 +73,27 @@ pub trait BabyLiminalExtension {
     ///
     /// The extension method ID matches the one declared in runtime:
     /// `BABY_LIMINAL_STORE_KEY_FUNC_ID`.
-    #[ink(extension = 41, returns_result = false)]
+    #[ink(extension = 41)]
```

Comment:
Attribute no longer existing in latest ink release.

Reply:
and the current behaviour is like `returns_result = false`?

Reply:
Looks like it from the examples give in the docs https://use.ink/macros-attributes/chain-extension/#attributes